README.md

```diff
@@ -9,7 +9,7 @@ Resources are simple memoized async functions that can run in any JavaScript run
 ```ts
 import alchemy from "alchemy";
 
-await using _ = alchemy("cloudflare-worker");
+await using Worker = alchemy("cloudflare-worker");
```

Comment:
`_` was intentional, but i'd otherwise accept `app` since this is the initialization of the "application scope", not a "Worker".